index.html

```diff
@@ -81,35 +81,14 @@
 			</div> -->
 				<!-- Slideshow container -->
 				<div class="slideshow-container">
-				  <div class="mySlides fade">
-				    <!-- <div class="numbertext">1 / 3</div> -->
-				    <img id="speakerPhoto1" class="photoS" src="">
-				    <!-- <div class="text">Caption Text</div> -->
-				  </div>
-
-				  <div class="mySlides fade">
-				    <!-- <div class="numbertext">2 / 3</div> -->
-				    <img id="speakerPhoto2" class="photoS" src="">
-				    <!-- <div class="text">Caption Two</div> -->
-				  </div>
-
-				  <div class="mySlides fade">
-				    <!-- <div class="numbertext">3 / 3</div> -->
-				    <img id="speakerPhoto3" class="photoS" src="">
-				    <!-- <div class="text">Caption Three</div> -->
-				  </div>
-
```

Comment:
don't need these anymore! we're going to let our js create them programatically.